cypress/integration/portal-dashboard/portal-dashboard-feedback.spec.js

```diff
@@ -255,6 +255,42 @@ context("Portal Dashboard Feedback Panel", () => {
       });
     });
   });
+  context('API Validation', () => {
+    beforeEach(() => {
+      cy.visit("/?portal-dashboard");
+      cy.get('[data-cy=navigation-select]').click();
+      cy.get('[data-cy="list-item-feedback-report"]').should('be.visible').click();
+      // Mock Firestore
+      cy.window().then((win) => {
+        win.firebase = {
+          firestore: () => ({
+            doc: () => ({
+              set: cy.stub().resolves()
+            })
+          })
+        };
+      });
+    });
+
+    it('displays "No feedback yet" when a student has no feedback', () => {
+      // Then try to access the activity-level feedback, using force: true since it might be disabled
+      cy.get('[data-cy=activity-level-feedback-button]')
+        .should('be.visible')
+        .click({force: true});
```

Comment:
Would it be a little better to make cypress take an action that ensures the button won't be disabled?